containers/hydra/postbuild.py

```diff
@@ -143,6 +153,11 @@ def main(argv: list[str]):
         # from Hydra web ui run command logs automatically.
         print(f'POSTBUILD_INFO="{nixbuildinfo}"')
 
+        themessage=nixbuildinfo
+        command = "/setup/messager.py -m "+themessage+"-f \"/setup/slack_config\"" 
```

Comment:
Path should not be in the docker image, but e.g. /home/hydra/slack_config (that's in the store / "volume" )
This path is mentioned several times in the script. Would be good to have just one place where it needs to be changed when we change it (it could be, e.g., one variable assignment)